File: OneCodeTeam/How to bind enum to RadioButtons in universal Windows apps/[C#]-How to bind enum to RadioButtons in universal Windows apps/C#/CSUnvsAppEnumRadioButton/CSUnvsAppEnumRadioButton.Shared/Common/BoolToStringConverter.cs
/****************************** Module Header ******************************\
 * Module Name:  BoolToStringConverter.cs
 * Project:      CSUnvsAppEnumRadioButton
 * Copyright (c) Microsoft Corporation.
 * 
 * This is converter of Boolean type and String type
 *  
 * 
 * This source is subject to the Microsoft Public License.
 * See http://www.microsoft.com/en-us/openness/licenses.aspx#MPL
 * All other rights reserved.
 * 
 * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
 * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
 * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
\***************************************************************************/

using System;
using Windows.UI.Xaml.Data;

namespace CSUnvsAppEnumRadioButton.Common
{
    public class BoolToStringConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, string language)
        {
            return (bool)value ? "Male" : "Female";
        }

        public object ConvertBack(object value, Type targetType, object parameter, string language)
        {
            string sex = value as string;

            if (sex == "Female")
                return false;
            return true;

        }
    }
}

File: OneCodeTeam/How to bind enum to RadioButtons in universal Windows apps/[C++]-How to bind enum to RadioButtons in universal Windows apps/C++/CppUnvsAppEnumRadioButton/CppUnvsAppEnumRadioButton.Shared/BoolToStringConverter.cpp
#include "pch.h"
#include "BoolToStringConverter.h"

using namespace CppUnvsAppEnumRadioButton::Common;

BoolToSexConverter::BoolToSexConverter()
{

}

BoolToSexConverter::~BoolToSexConverter()
{

}

Object^ BoolToSexConverter::Convert(Object^ value, TypeName targetType, Object^ parameter, String^ language)
{
	return (bool)value ? "Male" : "Female";
}

Object^ BoolToSexConverter::ConvertBack(Object^ value, TypeName targetType, Object^ parameter, String^ language)
{
	Platform::String^ sex = (Platform::String^)value;
	if (sex == "Female")
	{
		return false;
	}
	else
		return true;
}